lib/assessment/assess.py

```diff
@@ -13,7 +13,7 @@
 class KeyConceptError(Exception):
   pass
 
-def label(code, prompt, rubric, examples=[], api_key='', llm_model=DEFAULT_MODEL, num_responses=1, temperature=0.2, remove_comments=False, response_type='tsv', code_feature_extractor=None):
+def label(code, prompt, rubric, examples=[], api_key='', llm_model=DEFAULT_MODEL, num_responses=1, temperature=0.2, remove_comments=False, response_type='tsv', code_feature_extractor=None, lesson=None):
```

Comment:
I don't see changes in the flask app service code to get or pass the lesson through, is that intended to be a follow-up along with changes in the cdo dashboard code?

Reply:
Sorry, I mean that I see the lesson param getting passed through for the cde_test, but not /assessment. It makes sense for it to be a follow-up, I just wanted to check. Here's the relevant part of the cdo code: https://github.com/code-dot-org/code-dot-org/blob/b72a77895dfe6cd5a5c96a8b850d87f9c53394f9/dashboard/app/jobs/evaluate_rubric_job.rb#L331-L344

Reply:
Great callout Ceara! to make this work with rubric tester, you'll also need to pass the lesson param in here: https://github.com/code-dot-org/aiproxy/blob/76eb0b0b9f1ddef78963b078343ba9a00fb622e7/lib/assessment/rubric_tester.py#L254